testsuite/features/support/env.rb

```diff
@@ -73,6 +74,10 @@
 $beta_enabled = (ENV.fetch('BETA_ENABLED', 'False') == 'True')
 $api_protocol = ENV.fetch('API_PROTOCOL', nil) if ENV['API_PROTOCOL'] # force the API protocol to be used. You can use 'http' or 'xmlrpc'
 
+# Define a global counter to track consecutive ReadTimeout errors
+$timeout_failure_count = 0
```

Comment:
Better if we set this counter using the `add_context('timeout_failure_counter', 0)` and `get_context('timeout_failure_counter')`

Reply:
It fills over engineer. The variable is only use in this file.